pipe/psf_phot.py

```diff
@@ -699,7 +705,11 @@ def read_gain(self):
         """
         if self.pps.gain is None:
             self.mess('Reading HK and computing gain...')
-            self.pps.gain = np.median(gain(self.pps.file_hk, self.pps.file_gain))
+            gain_timeseries = gain(self.pps.file_hk, self.pps.file_gain)
+            if np.median(gain_timeseries) > 0.05:
+                self.pps.gain = np.median(gain_timeseries)
+            else:
+                self.pps.gain = 1.9

```

Comment:
This is a leftover circuit-breaker meant to catch badly initialized gains from the testing suite. (We shouldn't need this tweak – let me know if you think it makes sense to delete this now that the tests are running).